### ISSUE TITLE
Appliance schedules with unrealistic power

### ISSUE THREAD

**Issue by ejhw** [NREL/resstock#565]
| Max Power (kW)
-- | --
Clothes Dryer | 25.5252
Ceiling Fan | 0.0225
Clothes Washer | 1.385511
Dishwasher | 2.19534
Bath Fan | 0.022503
Whole Home Fan | 0
Range Fan | 0.030003
Plug Loads | 0.103389
Plug Loads | 0.288333
Range | 10.09916
Refrigerator | 0.065089

[Yesterday 3:41 PM] Maguire, Jeff
    The clothes dryer in particular in this home seems really high... But we're seeing other homes with max power over 30 kW and it seems largely due to some of the occupant schedules
​[Yesterday 3:41 PM] Maguire, Jeff
    since we're doing some buildings to grid cosim there's impacts on the grid model from having these spikes that might not cause any issues in a normal ResStock run
​[Yesterday 3:52 PM] Adhikari, Rajendra
    I think this is one limitation of the way we are handling the stochastic schedules. We scale the schedule (i.e. multiply by a certain factor) so that the annual energy matches with what we used to have before. So, if a certain house happens to have a very infrequent use of the dryer, they will end up having too high a power

[Yesterday 3:57 PM] Maguire, Jeff
    Do we plan to tackle this at all? It seems like it's going to keep coming up as we do buildings to grid cosimulation
​[Yesterday 3:58 PM] Maguire, Jeff
    My first impulse for these events with unrealistically high power is to reduce the max power and increase the duration for these events, but that's probably pretty tricky to do in practice
​[Yesterday 4:00 PM] Adhikari, Rajendra
    It's result of two conflicting goals: Honoring the schedule generated by the stochastic generator and honoring a realistic annual energy consumption. 
​[Yesterday 4:00 PM] Adhikari, Rajendra
    If the stochastic schedules were perfect, the peaks would be perfect too

[Yesterday 5:24 PM] Adhikari, Rajendra
    I think one approach worth exploring is allowing the annual energy consumption to be quite varied between households (so that we don't have to overly scale some buildings leading to huge peaks). As long as on average, annual energy consumption remains sensible, that should be fine. But yeah, that will require some time/budget allocation as well. 
(1 liked)

cc @jmaguire1 @rajeee

**Comment by rajeee** [comment]
@joseph-robertson  Yeah, think so. It hasn't been addressed and we shouldn't just forget about this!
